### DISCUSSION TITLE
translate and rotate part in Radioss

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1513] [Q&A]
Hi everyone,

I am new to Radioss. I am wondering if Radioss support any keywords that I can translate and rotate an part/instance like what I can do with *instance in ABAQUS? Thanks for any help

Best
Lee

**Comment by sebastienVilleneuve** [comment]
Hi @li2qn 

you may include your part in a //SUBMODEL ( cf https://help.altair.com/hwsolvers/rad/topics/solvers/rad/submodel_starter_r.htm#submodel_starter_r)
and translate/rotate this submodel , using /TRANSFORM option ( cf https://help.altair.com/hwsolvers/rad/topics/solvers/rad/transforms_overview_starter_r.htm#transforms_overview_starter_r)

I hope this helps
Regards,
Sébastien

**Reply by li2qn** [reply]
Thanks so  much for your help
